File: day 3/Task1.java
import java.util.Random;
import java.util.Scanner;
	public class Task1{
	 public static void main(String[] args){
Scanner input = new Scanner(System.in);
Random random = new Random();

   int random1 = random.nextInt(20);
   int random2 = random.nextInt(20);
int multiplicationRandom = random1 * random2;
int additionRandom = random1 + random2;
int subtractionRandom = random1 - random2;

   for(int count = 0; count <= 10; count++){
    if(multiplicationRandom < 13 && additionRandom > 10){
    System.out.printf("%s%d%s$d%s","what is ", random1, " * ", random2, " : "   );






}
}
}

  
}

File: day 3/python/task1.py
import random
correct_random_answer = 0 
random1 = random.random()
random2 = random.random()


for number in range(10):
	question = int(input(f"what is {random1}+ {random2}"))
random3 = random1 + random2
if question == random3:
	correct_random_answer = correct_random_answer + 1




print("you got" + correct_random_answer)



 